normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
```

Comment:
Usually I'd say mock up a fake one because: the schema may change, which would possibly change the test coverage. It also allows you to very specifically target aspects of the validation function to ensure it works correctly. But for now this should be fine. Maybe toss a TODO comment in to say we should probably do a more extensive test at some point.